runtime/lua/vim/lsp/codelens.lua

```diff
@@ -244,10 +250,7 @@ function M.refresh()
     return
   end
   active_refreshes[bufnr] = true
-  vim.lsp.buf_request(0, 'textDocument/codeLens', params, function (...)
-    active_refreshes[bufnr] = nil
-    vim.lsp.handlers['textDocument/codeLens'](...)
-  end)
+  vim.lsp.buf_request(0, 'textDocument/codeLens', params, M.on_codelens)
```

Comment:
To make sure `vim.lsp.codelens.on_codelens()` is called, I figured it'd make sense to explicitly pass it as the handler here? I can't think of a use case where a custom `textDocument/codeLens` handler should be called when calling `vim.lsp.codelens.refresh()`.